config/routes.rb

```diff
@@ -21,7 +21,9 @@ def matches?(request)
 
     constraints account: /[^\/\?]+/ do
       constraints authenticator: /authn-?[^\/]*/, id: /[^\/\?]+/ do
-        get '/:authenticator/:service_id/:account/status' => 'authenticate#status'
+        get '/:authenticator(/:service_id)/:account/status' => 'authenticate#status'
```

Comment:
Why is `service_id` optional? do we have a status check for `authn`?

Reply:
@micahlee do you remember why we did this? IIRC it was a bug we discovered during the hackathon but I can't recall the details.

Reply:
Yes, it's because the built-in authenticator is just `authn` without an additional `/:service_id` component.

Reply:
sure, but why would someone want to know the status of the `authn` authenticator? isn't it always working?

The purpose of the `status` endpoint is for the operator to validate that the authenticator is configured properly, to get an indication that users can authenticate against it. it doesn't make much sense to have that for `authn`.

Having said that, if the `status` check for authn passes then it's fine. I didn't check that when we developed the status check because i didn't see any need to. If it works and doesn't fail on some weird error that assumes `service-id` is not blank then this change is fine. Please validate this before making this change.

Reply:
Going to remove this change because YAGNI. I'd be surprised if there was a reason we had to do this and it may have just been for API consistency.